pallets/maintenance-mode/Cargo.toml

```
@@ -0,0 +1,27 @@
+[package]
+name = "pallet-maintenance-mode"
+version = "0.1.0"
+authors = ["PureStake"]
+edition = "2018"
+description = "Puts a FRAME-based runtime into maintenance mode where restricted interactions are allowed."
+
+[dependencies]
+frame-support = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.4", default-features = false }
+frame-system = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.4", default-features = false }
+log = "0.4"
+sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.4", default-features = false }
+parity-scale-codec = { version = "2.0.0", default-features = false }
+
+[dev-dependencies]
+sp-io = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.4", default-features = false }
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.4", default-features = false }
```

```suggestion
sp-io = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.4" }
sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.4" }
```

We learned from Elois that tests are std anyway so we don't need `default-features = false` nor the the `std` features for these deps further below.